components/gene-panels/api/src/main/java/org/phenotips/panels/internal/PatientDataAdapter.java

```diff
@@ -120,7 +120,7 @@ private PatientDataAdapter(@Nonnull final AdapterBuilder builder)
         private static final String REJECTED = "rejected";
 
         /** The internal {@link PatientData} label for rejected candidate genes. */
-        private static final String REJECTED_CANDIDATE = "rejected_candidate";
+        private static final String REJECTED_CANDIDATE = "rejectedcandidate";
```

Comment:
Renaming this value seems to trigger way too many changes. The reason why I renamed it in the draft code that I originally proposed for the UI is this: the velocity script for generating the hierarchical layout of the status options relies on the convention that if, for example, a status value starts with `candidate_`, then it is a descendant of `candidate`. `rejected_candidate` was likely renamed as a quick fix to prevent displaying it as a descendant of `rejected`. Renaming already existing values can be easily avoided by choosing a different separator instead of `_` to mark descendants, for example `>`: `candidate>no el_disease`, `umc_vus`, etc.